samples/net/wifi/boards/rd_rw612_bga.conf

```diff
@@ -73,6 +73,8 @@ CONFIG_ZPERF_WORK_Q_THREAD_PRIORITY=3
 CONFIG_NET_SOCKETS_SERVICE_THREAD_PRIO=3
```

Comment:
In commit `subsys: net: Increase net_mgmt task priority`, please remove the `subsys:` prefix as that is not needed to mention.

Reply:
Removed `subsys:` in commit message.